stock_change_qty_reason/views/stock_inventory_line_view.xml

```diff
@@ -24,28 +24,4 @@
             </field>
         </field>
     </record>
-    <record id="reason_stock_inventory_line_tree2" model="ir.ui.view">
-        <field name="name">stock.inventory.line.tree</field>
-        <field name="model">stock.inventory.line</field>
-        <field name="inherit_id" ref="stock.stock_inventory_line_tree2" />
-        <field name="arch" type="xml">
-            <field name="location_id" position="before">
-                <field name="reason" />
```

Comment:
What has motivated this change? This view doesn't exist anymore on v14?

Reply:
Exactly, the two views that end with tree2, they don't exist anymore on version 14. So, as everything now is displayed on the other two views, I deleted the other two.

Reply:
thank you!